docker-compose.ci.yml

```diff
@@ -15,15 +15,3 @@ services:
     build:
       context: ./webapp
       target: cron
-
-  erica:
-    image: steuerlotse_erica:${DOCKER_TAG:-latest}
-    build:
-      context: ./erica_app
-      args:
-        bucket_name: ${ERICA_BUCKET_NAME}
-        access_key_id: ${ACCESS_KEY_ID}
-        access_key: ${ACCESS_KEY}
-        endpoint_url: ${ENDPOINT_URL}
-        elster_datenlieferant: ${ELSTER_DATENLIEFERANT}
-        elster_hersteller_id: ${ELSTER_HERSTELLER_ID}
```

Comment:
#635 would enable us to run docker compose locally to get the services running. We probably need to change/move that, too.

Reply:
I commented on that PR and Jördis knows what's coming :)